specification/communication/Communication.JobRouter/tspconfig.yaml

```diff
@@ -0,0 +1,53 @@
+parameters:
+  "python-sdk-folder":
+    default: "{project-root}/azure-sdk-for-python/"
+  "java-sdk-folder":
+    default: "{project-root}/azure-sdk-for-java/"
+  "js-sdk-folder":
+    default: "{project-root}/azure-sdk-for-js/"
+  "csharp-sdk-folder":
+    default: "{project-root}/azure-sdk-for-net/"
+  "service-directory-name":
+    default: "communication"
+  "service-dir":
+    default: "sdk/communication"
+  "python-output-dir":
+    default: "azure/communication/jobrouter/_generated"
+emit: [
+  "@azure-tools/typespec-autorest"
+]
+
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    output-file: "{azure-resource-provider-folder}/JobRouter/{version-status}/{version}/communicationservicejobrouter.json"
+    emitter-output-dir: "{project-root}/.."
+    omit-unreachable-types: true
+    examples-directory: "{project-root}/examples"
+  "@azure-tools/typespec-csharp":
+    clear-output-folder: true
+    model-namespace: false
+    emitter-output-dir: "{csharp-sdk-folder}/sdk/{service-directory-name}/{namespace}/src"
+    namespace: Azure.Communication.JobRouter
+    package-dir: "Azure.Communication.JobRouter"
+    save-inputs: true
+  "@azure-tools/typespec-python":
+    package-pprint-name: "\"Communication JobRouter\""
+    emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
+    package-dir: "azure-communication-jobrouter"
+    package-name: "{package-dir}"
+    package-version: 1.0.0
+  "@azure-tools/typespec-java":
+    emitter-output-dir: "{java-sdk-folder}/sdk/{service-directory-name}/azure-communication-jobrouter"
+    namespace: com.azure.communication.jobrouter
+    partial-update: true
+  "@azure-tools/typespec-ts":
+    emitter-output-dir: "{js-sdk-folder}/sdk/{service-directory-name}/communication-job-router-rest"
+    package-dir: "communication-job-router"
```

Comment:
I don't think we should specify the `emitter-output-dir` here because we use the option `package-dir`. And it should be ${folder-name}-rest.  See name [convention](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/RLC-quickstart.md#structure-your-project) here.

Could you update with below?
```
package-dir: "communication-job-router-rest"
```